CommandLine.Tests/CommandTests.cs

```diff
@@ -158,6 +158,21 @@ public void ParseResult_Command_identifies_innermost_command()
         }
 
         [Fact]
+        public void ParseResult_Command_identifies_implicit_root_command()
+        {
+            var parser1 = new Parser(
+                Option("-x", ""),
+                Option("-y", ""));
+
+            var result = parser1.Parse("-x -y");
+
+            var command = result.Command();
+
+            command.Should().NotBeNull();
+            command.Name.Should().Be("root");
```

Comment:
We probably want to identify the actual executable name rather than use "root" here.